distributions/aws/test/e2e/tests/test_sanity.py

```diff
@@ -0,0 +1,127 @@
+"""
+Installs the vanilla distribution of kubeflow and validates the installation
```

Comment:
Can you expand this to describe how we are validating

Reply:
done